File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/C/progam1.c

#include <stdio.h>
#include <stdlib.h>

int main() {

	int n;
	printf("Enter Number : ");
	scanf("%d", &n);
	
	if(n <= 0)
		exit(0);

	for(int i = 1; i <= n*n; i= i + n){
		printf("%d ", i);
	}
	printf("\n");
}

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/Java/progam1.java
import java.io.*;

class Template {
	public static void main(String[] args) throws IOException {
		
		int n = 0;
		System.out.printf("Enter Number : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		try {
			n = Integer.parseInt(br.readLine());
		}catch(NumberFormatException nf){
			System.exit(0);
		}

		if(n <= 0)
			System.exit(0);

		for(int i = 1; i <= n*n; i= i + n){
			System.out.printf("%d ", i);
		}
		System.out.printf("\n");
	}
}
